the-collab-lab/tcl-16-smart-shopping-list
# Add sass as a dependency and create a scss file with our global variables and mixins

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



Added sass as a dependency and updated code to include a base.scss file which includes styles to be used globally (ie. headings, body text, inputs and button styles). The variable.scss includes some common variables and mixins and normalize.scss is added to render all elements consistently across browers. These global styles are a work in progress - this issue is mainly to ensure global variables and sass have been added to the repo. 

## Related Issue


closes #31

## Acceptance Criteria


- add sass into project as a dependency
- create a variable.scss file that will hold common reusable variables outlined in our wireframes/mood board: https://www.notion.so/Mood-Board-5f62f8cd9a5a45359a32dd7b901151f5

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


N/A

### After


N/A


## Testing Steps / QA Criteria


- pull from branch
- npm install new dependency 
- ensure sass loads properly and you don't get any sass error messages 
- check to see if fonts are loading properly by inspecting the elements (body should be merriweather and headings should be lato)
